ansible/playbooks/utils/check_volume.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Checking for necessary --extra_vars
+  hosts: all
+  vars:
+    example:
+      - "-e \"{'VOLUME_DEVICE':'/vol1', 'VOLUME_MKFS_DEVICE':True, 'VOLUME_DEVICE_TYPE': 'ext4'}\""
+
+  tasks:
+
+    - fail: msg="Must set VOLUME_DEVICE. Pass a value in via `-e` as JSON {{example[0]}}"
```

Comment:
Ansible doesn't care how a variable is passed (whether via extra_vars or any other way), and besides, I don't think that is JSON, it's YAML ["inline hash" format](http://yaml.org/YAML_for_ruby.html#simple_inline_hash). So, I would just say "Must set VOLUME_DEVICE". (Same applies everywhere else in this PR that you mention JSON).

Also, why do you make `example` a list and then index into its first element, when it could just be a string?